src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -171,6 +171,8 @@
                             var text = requestStack.length + ' AJAX request' + (requestStack.length > 1 ? 's' : '');
                             infoSpan.textContent = text;
                         }
+
+                        ajaxToolbarPanel.style.display = 'block';
```

Comment:
I suggest using: `ajaxToolbarPanel.style.display = null`
This only removes the display style, so the original CSS can kick in.